paypal/standard/helpers.py

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 import hashlib
-
+import sys
 from django.conf import settings
 from django.utils.encoding import smart_str
-
 from paypal.utils import warn_untested
 
+if sys.version_info[0] >= 3:
+    unicode = str
```

Comment:
This seems a bit hacky - on Python 3 `unicode` doesn't exist, and we should upgrade the main code itself to work correctly as normal Python 3 code. This makes things simpler when it comes to dropping Python 2 compat.

Probably we should be using `six.text_type` in the places that need `unicode`.